ossm/values.yaml

```diff
@@ -100,7 +100,6 @@ csv:
     * [Documentation](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/latest/html/about/ossm-about-openshift-service-mesh)
     * [Bugs](https://issues.redhat.com/projects/OSSM)
   support: Red Hat, Inc.
-  version: 3.1.2
```

Comment:
Why is this removed?

Reply:
It's actually used in HELM_TEMPL_DEF_FLAGS but later overwritten during bundle creation in BUNDLE_GEN_FLAGS. So this is a bad design but please keep it there with correct version and I will try to fix this confusing behavior upstream.